pkg/core/helper_test.go

```diff
@@ -395,6 +396,7 @@ func initBasicChain(t *testing.T, bc *Blockchain) {
 		},
 	}
 	require.NoError(t, addNetworkFee(bc, transferTx, acc0))
+	transferTx.SystemFee += 1000000
```

Comment:
Why? We already have `11000000` system fee and the change here shouldn't affect the amount consumed for simple transfer.

Reply:
Let me see, not fixed yet, but the previously set amount isn't enough.

Reply:
It's connected with onNEP17Payment method added to a contract (which is the sender).

Reply:
Got this, I've added the contract's owner:
```
var owner = util.FromAddress("NbrUYaZgyhSkNoRo9ugRyEMdUZxrhkNaWB")
```
so it needs to be initialised before transfer invocation. Previosly set amount is not enough for initialisation.